src/mlpack/core/tree/hollow_ball_bound.hpp

```diff
@@ -213,6 +221,21 @@ struct BoundTraits<HollowBallBound<MetricType, ElemType>>
   const static bool HasTightBounds = false;
 };
 
+namespace meta {
+
+template<typename BoundType>
+struct IsHollowBallBound
+{
+  static const bool value = false;
+};
+
+template<typename MetricType, typename ElemType>
+struct IsHollowBallBound<HollowBallBound<MetricType, ElemType>>
+{
+  static const bool value = true;
+};
+
```

Comment:
yeah, I agree. I thought it will be useful, but then I realized that I needn't it.